src/styles/components/alert/styles.less

```diff
@@ -1,7 +1,7 @@
 & when (@alert-enabled) {
 
   .alert {
-    align-items: center;
+    align-items: flex-start;
```

Comment:
This is supported by IE11 and Edge, and we actually use this quite a bit throughout the codebase. So we're good to go!